pandas/io/stata.py

```diff
@@ -164,6 +165,7 @@ def read_stata(filepath_or_buffer, convert_dates=True,
                convert_missing=False, preserve_dtypes=True, columns=None,
                order_categoricals=True, chunksize=None, iterator=False):
 
+    filepath_or_buffer = os.path.normpath(filepath_or_buffer)
```

Comment:
This should be put in the common io logic, not special cased for stata, likely here
https://github.com/pandas-dev/pandas/blob/f11bbf2f505d81900cc83ce387a6a1b1d2a2f866/pandas/io/common.py#L148 @